src/components/atoms/BottomMenu/BottomMenu.tsx

```diff
@@ -116,6 +129,56 @@ export const BottomMenu = memo(
         );
       }, []);
 
+      const renderMenuHeader = () => {
+        if (!header) {
+          return null;
+        }
+
+        const {title, subtitle, onBackPress, onClosePress, closeButtonVisible} =
+          header;
+
+        const buttonsVisible =
+          onBackPress || onClosePress || closeButtonVisible;
+        return (
+          <View>
+            {buttonsVisible ? (
+              <View style={styles.headerButtonsContainer}>
+                {onBackPress ? (
+                  <AnimatedCircleButton
+                    icon={{
+                      name: 'chevronMediumLeft',
+                    }}
+                    testID={composeTestID(testID, 'backButton')}
+                    onPress={hide}
+                  />
+                ) : (
+                  <View />
+                )}
+                {closeButtonVisible || onClosePress ? (
+                  <AnimatedCircleButton
+                    icon={{
+                      name: 'close',
+                    }}
+                    testID={composeTestID(testID, 'closeButton')}
+                    onPress={onClosePress || hide}
+                  />
+                ) : (
+                  <View />
+                )}
+              </View>
+            ) : null}
+            {title ? (
+              <View style={styles.headerTextContainer}>
+                {title ? <Text style={styles.headerTitle}>{title}</Text> : null}
+                {subtitle && title ? (
```

Comment:
`title` isn't necessary here. You checked it above.
```suggestion
                <Text style={styles.headerTitle}>{title}</Text>
                {subtitle ? (
```